scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/synchronization/SynchronizationWrapper.java

```diff
@@ -0,0 +1,248 @@
+            throws InvalidChannelException, IOException {
+        return internalAPI.replace(originator, taskId, channel, key, value);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfAbsent(String channel, String key, String mappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfAbsent(originator, taskId, channel, key, mappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T computeIfPresent(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.computeIfPresent(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T compute(String channel, String key, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.compute(originator, taskId, channel, key, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> T merge(String channel, String key, T value, String remappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.merge(originator, taskId, channel, key, value, remappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> PredicateActionResult<T> conditionalCompute(String channel, String key,
+            String predicate, String thenRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.conditionalCompute(originator, taskId, channel, key, predicate, thenRemappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> PredicateActionResult<T> conditionalCompute(String channel, String key,
+            String predicate, String thenRemappingFunction, String elseRemappingFunction)
+            throws InvalidChannelException, CompilationException, IOException {
+        return internalAPI.conditionalCompute(originator,
+                                              taskId,
+                                              channel,
+                                              key,
+                                              predicate,
+                                              thenRemappingFunction,
+                                              elseRemappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> void conditionalReplaceAll(String channel, String predicate,
+            String thenRemappingFunction) throws InvalidChannelException, CompilationException, IOException {
+        internalAPI.conditionalReplaceAll(originator, taskId, channel, predicate, thenRemappingFunction);
+    }
+
+    @Override
+    public <T extends Serializable> void conditionalReplaceAll(String channel, String predicate,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') T is not used in the method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2326)